### PR TITLE
Compiler error when using instance members on static class

### PR DESCRIPTION
**Pull request by edgarfgp**
Fixes one the multiple missing error reporting in https://github.com/dotnet/fsharp/issues/13147

### CODE REVIEW

**Review by KevinRansom — APPROVED** [review]
Thanks, looks good.

File: tests/FSharp.Compiler.ComponentTests/Language/StaticClassTests.fs [hunk 1]
```diff
@@ -201,4 +201,55 @@ type B =
         Fs code
         |> withLangVersion langVersion
         |> compile
-        |> shouldSucceed
\ No newline at end of file
+        |> shouldSucceed
+
+    [<Fact>]
+    let ``Sealed and AbstractClass on on a type with instance members in lang version70`` () =
```
Comment (psfinaki) [inline tests/FSharp.Compiler.ComponentTests/Language/StaticClassTests.fs:207]: Double "on"

File: src/Compiler/FSComp.txt [hunk 2]
```diff
@@ -1668,3 +1668,4 @@ featureEscapeBracesInFormattableString,"Escapes curly braces before calling Form
 3551,buildDuplicateFile,"The source file '%s' (at position %d/%d) already appeared in the compilation list (at position %d/%d). Please verify that it is included only once in the project file."
 3552,chkConstructorWithArgumentsOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Constructor with arguments is not allowed."
 3553,chkAdditionalConstructorOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Additional constructor is not allowed."
+3554,chkInstanceMemberOnStaticClasses,"If a type uses both [<Sealed>] and [<AbstractClass>] attributes, it means it is static. Instance member is not allowed."
```
Comment (psfinaki) [inline src/Compiler/FSComp.txt:1671]: "Instance members are not allowed"

**Review by psfinaki — DISMISSED** [review]
A few grammar remarks, otherwise all good

**Review by KevinRansom — APPROVED** [review]

**Review by T-Gro — APPROVED** [review]

**Pull request merged**